### DISCUSSION TITLE
இந்த model எப்படி ட்ரைனிங் செய்யப்பட்டது

### DISCUSSION THREAD

**Discussion by anandav** [gnana70/tamil_ocr#47] [General]
Hi ஞான பிரசாத்
I don't know anything about ML , your project looks very interesting. Best compared to some online ocr's.
 I like know how do you trained this model  to get craft_mlt_25k.pth and parseq_tamil.pt 

Thanks

**Comment by anandav** [comment]
I haven't received a response from the dev. As a result, I'm closing this for now.

**Comment by gnana70** [comment]
Hi @anandav , I haven't trained CRAFT model as it's openly available. I have trained the parseq_tamil.pt with image-text pairs in tamil language
